src/internal/m365/graph/service.go

```diff
@@ -63,18 +71,18 @@ type Servicer interface {
 var _ Servicer = &Service{}
 
 type Service struct {
-	adapter *msgraphsdkgo.GraphRequestAdapter
+	adapter abstractions.RequestAdapter
```

Comment:
Quick sanity check on this: Is the removal of pointer intentional here? Looks fine to me, but just want to confirm